osbs/build/user_params.py

```diff
@@ -412,16 +486,21 @@ def set_params(self,
         self.yum_repourls.value = yum_repourls or []
         self.compose_ids.value = compose_ids or []
 
+        variations = (self.scratch.value, self.is_auto.value, self.isolated.value)
+        if variations.count(True) > 1:
```

Comment:
Can you please do
```
if any(self.scratch.value, self.is_auto.value, self.isolated.value):
```
instead?

Reply:
Sure.  I was just copying existing code but your version is much better.

Reply:
except having none of them set is fine (so not any()) and have two of them set is not.